hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/shell/Handler.java

```diff
@@ -118,6 +119,29 @@ protected void printObjectAsJson(Object o) throws IOException {
     out().println(JsonUtils.toJsonStringWithDefaultPrettyPrinter(o));
   }
 
+  protected String objectToJsonString(Object o) throws IOException {
+    return JsonUtils.toJsonStringWithDefaultPrettyPrinter(o);
+  }
+
+  /**
+   * Print the results in the iterator as a valid JSON array to out().
+   * @return Number of entries actually printed.
+   */
+  protected int printAsJsonArray(Iterator<?> iterator, int limit)
+      throws IOException {
+    int counter = 0;
+    out().print("[ ");
+    while (limit > counter && iterator.hasNext()) {
+      if (counter > 0) {
+        out().print(", ");
+      }
+      out().print(objectToJsonString(iterator.next()));
+      counter++;
+    }
+    out().println(" ]");
```

Comment:
Can we build the list using jackson in `JsonUtils` instead of mixing library generated json with manually generated json? JsonUtil has a `toJsonList` function but its actually a deserializer, not a serializer as the name implies. It also looks unused. Maybe we can make to `toJsonList` actually serialize a list of objects and use that here instead.

Reply:
Yup. Great idea. I should have done this in the first place.